recipes/cxxtest/cxxtestgen

```diff
@@ -0,0 +1,20 @@
+#!/opt/anaconda1anaconda2anaconda3/bin/python
+#
+# The CxxTest driver script, which uses the cxxtest Python package.
```

Comment:
Should this be committed as part of the recipe or is this coming from the source code? If the latter, we should probably drop it. If the former, it would be good to understand what this is needed for.